src/components/Todo.jsx

```diff
@@ -1,19 +1,26 @@
 import React from 'react';
+import { EditTodoForm } from './EditTodoForm';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 import { faPenToSquare } from '@fortawesome/free-regular-svg-icons';
 import { faTrash } from '@fortawesome/free-solid-svg-icons';
 import { faCircle, faCheckCircle } from '@fortawesome/free-regular-svg-icons';
 import { useSortable } from '@dnd-kit/sortable';
 import { CSS } from '@dnd-kit/utilities';
+import PropTypes from 'prop-types';
 
 import './Todo.css';
 
-export const Todo = ({task, toggleComplete, deleteTodo, editTodo}) => {
-  const [bgColor, setBgColor] = React.useState('#0a9396');
+const colors = {
+  primary: '#0a9396',
+  secondary: '#136466',
+};
```

Comment:
minor thing, just to mention that is normal to hace these two concepts around: the primary color and the seconday color